src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java

```diff
@@ -33,12 +34,15 @@
 
     private final DwpAddressLookupService dwpAddressLookupService;
     private final AirLookupService airLookupService;
+    private final boolean enhancedConfidentialityFeature;
 
     @Autowired
     public RoboticsJsonMapper(DwpAddressLookupService dwpAddressLookupService,
-                              AirLookupService airLookupService) {
+                              AirLookupService airLookupService,
+                              @Value("${feature.enhancedConfidentiality.enabled}") boolean enhancedConfidentialityFeature) {
```

Comment:
default it to false, like https://github.com/hmcts/sscs-pdf-email-common/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/service/PdfStoreService.java#L37